.github/workflows/ci.yml

```diff
@@ -8,63 +8,34 @@ on:
   push:
     branches:
       - main
+      - cross
     paths-ignore:
       - '**/*.md'
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
   cancel-in-progress: ${{ github.ref_name != 'main' }}
 
 jobs:
-  check:
-    name: Check
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - os: windows-latest
-          - os: ubuntu-latest
-          - os: macos-latest
-    runs-on: ${{ matrix.os }}
```

Comment:
I think you want to keep this, right?

Reply:
Oh yes ... I was hazy due to the outdates.